src/components/ArticleContainer.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect} from 'react'
+import Spinner from 'react-bootstrap/Spinner';
+import axios from 'axios'
+import ArticleCard from './ArticleCard';
+import Row from 'react-bootstrap/Row';import Col from 'react-bootstrap/Col';
+
+
+
+function ArticleContainer() {
+    const [articles, setArticles] = useState([]);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        axios.get('https://nc-news-2.onrender.com/api/articles')
+        .then((response) => {
+            setArticles(response.data.articles);
+            setLoading(false)
+        })
+        .catch((error) => {
+            console.log(error);
+        })
+    }, [])
+
+    if (loading){
```

Comment:
This is great user feedback utilising a loading state. I really like the spinner!